helm_deploy/values-dev.yaml

```diff
@@ -10,20 +10,21 @@ generic-service:
 
   env:
     ENVIRONMENT: dev
-    INGRESS_URL: "https://community-accommodation-tier-2-dev.hmpps.service.justice.gov.uk"
-    APPROVED_PREMISES_API_URL: "https://approved-premises-api-dev.hmpps.service.justice.gov.uk"
-    HMPPS_AUTH_URL: "https://sign-in-dev.hmpps.service.justice.gov.uk/auth"
-    TOKEN_VERIFICATION_API_URL: "https://token-verification-api-dev.prison.service.justice.gov.uk"
+    INGRESS_URL: 'https://community-accommodation-tier-2-dev.hmpps.service.justice.gov.uk'
+    APPROVED_PREMISES_API_URL: 'https://approved-premises-api-dev.hmpps.service.justice.gov.uk'
+    HMPPS_AUTH_URL: 'https://sign-in-dev.hmpps.service.justice.gov.uk/auth'
+    TOKEN_VERIFICATION_API_URL: 'https://token-verification-api-dev.prison.service.justice.gov.uk'
     AUDIT_LOG_ERRORS: 'true'
     COMMUNITY_ACCOMMODATION_API_TIMEOUT_RESPONSE: 30000
     COMMUNITY_ACCOMMODATION_API_TIMEOUT_DEADLINE: 30000
+    ADD_NOTES_DISABLED: 'false'
```

Comment:
does this not mean the e2e's will fail, if the flag is off in dev?

Reply:
ummmm, good point that the flag has a confusing name. But if it's false that notes are disabled in dev, then they are enabled in dev, so the e2es should pass, right?